org.adempiere.server-feature/setup/configuration/config.ini

```diff
@@ -107,7 +107,8 @@ osgi.bundles=org.apache.felix.scr@1:start,\
  org.osgi.util.xml,\
  org.postgresql.jdbc,\
  com.zaxxer.HikariCP,\
- wrapped.com.oracle.database.jdbc.ojdbc10,\
+ wrapped.com.oracle.database.jdbc.ojdbc17,\
+ wrapped.com.oracle.database.xml.xdb,\
```

Comment:
hi @CarlosRuiz-globalqss , I'm not seeing the reference to this jar in other configuration file.